### PR TITLE
Disallow access to the my-projects route if not logged in

### LINKED ISSUE #683: Disallow access to protected routes if unauthorized

### PR DESCRIPTION
**Pull request by allthesignals**
Closes #683

### CODE REVIEW

File: app/controllers/show-geography.js [hunk 1]
```diff
@@ -159,6 +159,8 @@ export default class ShowGeographyController extends GeographyParachuteControlle
       this.transitionToRoute('oops');
     }
 
+    console.log(projects);
```
Comment (godfreyyeung) [inline app/controllers/show-geography.js]: oops, should remove

File: app/routes/my-projects.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
-export default class MyProjectsRoute extends Route {
+const AuthenticatedRoute = Route.extend(AuthenticatedRouteMixin);
```
Comment (godfreyyeung) [inline app/routes/my-projects.js]: will we also need to add this to app/routes/show-project.js?

File: app/controllers/show-geography.js (hunk 1 above)
Comment (allthesignals) [reply app/controllers/show-geography.js]: thx

File: app/routes/my-projects.js (hunk 2 above)
Comment (allthesignals) [reply app/routes/my-projects.js]: @godfreyyeung nope — you don't need to be authenticated to look at the whole route. You only need to be authenticated to see the buttons for hearings/rec. Right?

**Pull request merged**